infra/README.md

```diff
@@ -6,10 +6,14 @@ Ensure that you have `terraform` and `gcloud` installed locally.
 
 Run `make gcloud-login` to login using `gcloud` if you are not logged in already.
 
+### Using Terraform
+
+Make sure you have the appropriate environment variables set using `source .env.[environment]` from the root folder.
```

Comment:
Is this still accurate? IIRC we wanted any `source`-ing (if any) to happen in the `make` recipe using the `sh -c` trick.